vizro-core/docs/pages/user-guides/custom-css.md

```diff
@@ -314,6 +314,14 @@ It's essential to understand the relationship between the targeted CSS class or
 
 ## Common examples
 
+### Make your CSS responsive to theme Switches with variables
```

Comment:
Should `Switches` be capitalized? @stichbury

Reply:
Ups, no, that's a typo